tabletop/site_sections/tabletop_tournaments.py

```diff
@@ -44,6 +44,19 @@ def sign_up(self, session, tournament_id, attendee_id, cellphone):
                 'state': _state(session)
             }
 
+    @ajax
+    def drop(self, session, attendee_id, tournament_id):
+        try:
+            session.delete(session.tabletop_entrant(attendee_id=attendee_id, tournament_id=tournament_id))
+            session.commit()
+        except:
+            log.error('unable to drop tournament entrant', exc_info=True)
+
+        return {
+            'message': 'Entrant dropped; if re-added they will be re-texted',
+            'state': _state(session)
+        }
```

Comment:
Rick wanted the ability to drop people from tournaments who say they cannot make it.